clients/banking/src/components/CardItemPhysicalDetails.tsx

```diff
@@ -800,15 +831,21 @@ export const CardItemPhysicalDetails = ({
 
                         <View style={styles.spendingContainer}>
                           <View style={styles.spendingLimitText}>
-                            <LakeText color={colors.gray[800]} variant="smallRegular">
+                            <LakeText
+                              color={bindingUserError ? colors.gray[300] : colors.gray[800]}
```

Comment:
As there are several time this ternary `bindingUserError ? colors.gray[300] : colors.gray[800]` maybe we could create a variable to store this.  
I don't know what will be the best name but I think something similar to `textColor`.  
I think it will help for maintenability without impacting readability.  
What do you think?